docs/Theming.md

```diff
@@ -1,6 +1,6 @@
 # Theming Ignite Apps
 
-[Back to README][./README.md]
+[Back to README][./readme.md]
```

Comment:
linter/formatter did this. not sure if matters

Reply:
Yeah it did that to me in #2037 as well